app/src/main/AndroidManifest.xml

```diff
@@ -164,8 +164,14 @@
             android:parentActivityName=".dashboard.DashboardActivity" />
         <activity
             android:name=".mycourse.player.VideoPlayerActivity"
+            android:taskAffinity=".mycourse.player.VideoPlayerActivity"
+            android:parentActivityName=".mycourse.MyCourseActivity"
+            android:autoRemoveFromRecents="true"
```

Comment:
Why have you added both of them i.e exclude and autoRemove ?

Reply:
Sir because we don't want to show it in recent tasks list

Reply:
Sir I got the ans in https://medium.com/mindorks/android-picture-in-picture-mode-and-backstack-management-for-multiple-activities-8427fe3f8102 about that backpressed problem

Reply:
https://stackoverflow.com/questions/44363530/difference-between-autoremovefromrecents-and-excludefromrecents
first of all, not both of them will be required and for instance and secondly take a look at youtube or google maps implementation of PIP

Reply:
So should I keep only one?